scripts/run-swift-format.sh

```diff
@@ -39,14 +39,15 @@ swiftformat_bin=${swiftformat_bin:-$(command -v swift-format)} || fatal "❌ swi
 "${swiftformat_bin}" lint \
   --parallel --recursive --strict \
   "${repo_root}/Sources" "${repo_root}/Tests" \
+    "${repo_root}/benchmarks/Benchmarks" "${repo_root}/benchmarks/Sources" "${repo_root}/benchmarks/Tests" \
```

Comment:
Can't we just format `${repo_root}/benchmarks/`?

Reply:
I think that should work. Will also format the `Package.swift` then

Reply:
Just tested and it doesn't work because it will also try to format hidden folders and therefore our checked out dependencies e.g.:
```
~/Repositories/swift-format/.build/release/swift-format format --parallel --recursive --in-place ./                                                                     
/Users/davidnadoba/Repositories/swift-certificates/benchmarks/.build/5.8/checkouts/package-benchmark/Sources/Benchmark/Benchmark.swift:18:46: error: expected declaration after attribute in conditional compilation clause
```

Reply:
Yeah I think what I would recommend is inverting this (don't know if `swift-format` supports that) e.g. passing excludes instead of includes

Reply:
It doesn't support that